lib/plug/logger.ex

```diff
@@ -41,6 +42,15 @@ defmodule Plug.Logger do
     end)
   end
 
+  # TODO: remove this once Plug supports only Elixir 1.2.
+  if function_exported?(:erlang, :monotonic_time, 0) do
+    defp current_time, do: :erlang.monotonic_time
```

Comment:
Let's not do that because we lose precision. The ideal is to always convert later.